packages/snap/src/rpc/setBirthdayBlock.tsx

```diff
@@ -0,0 +1,48 @@
+import { Form, Box, Heading, Input, Button, Text } from "@metamask/snaps-sdk/jsx";
+
+type BirthdayBlockForm = {customBirthdayBlock: number | null}
```

Comment:
Please reformat files with prettier

Reply:
Lint is currently broken across the snap and web-wallet, will address fixing linter to @chainsafe/eslint-config in separate issue